CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# 🎇Contributing Guidelines
+
+This documentation contains a set of guidelines to help you during the contribution process. 
+
+I'm happy to welcome all the contributions from anyone willing to add new scripts to this repository. Thank you for helping out and remember,
+**no contribution is too small.**
+
+
+## 💻Before Contributing
+
+Welcome to [arpitghura/typing-test](https://github.com/arpitghura/typing-test/). Before sending your pull requests, make sure that you **read the whole
+guidelines**. If you have any doubt on the contributing guide, please feel free to reach out.
+
+
+## 🙌Contribution
+
+Any contribution is accepted, from fixing grammatical mistakes to implementing complex scripts. Please read this section if you are contributing your work.
+
+
+### 🔖Steps to Contribute
+
+Following are the steps to guide you:
+
+* Step 1: Fork the repo and Go to your Git terminal and  clone it on your machine.
+* Step 2: Add a upstream link to main branch in your cloned repo
```

Comment:
Add the simple method too which has following steps:
1. Fork into your account
2. Clone the repo in your machine using `git clone https://github.com/<your_username>/typing-test.git`
3. Replace the `<your_username>` with your github username.
4. then create a new branch and so on.